WebRoot/skins/_base/base4/skin.properties

```diff
@@ -2322,3 +2322,19 @@ PrefOOOExternalMsg          = margin-top:@PrefViewGutterSpace@;
 PrefOOOTimePeriodCell       = padding:@PrefSectionInternalSpacing@ 5px;
 PrefOOOMiniDropDown         = @PrefRowMiniSelectWidget@;
 PrefOOODurationLabel        = text-transform:lowercase;
+
+######################################
+#   PREFERENCES -Mail Page
+######################################
+
+PrefMailComposeRowValueWrapperWidth            = 22%
+PrefMailBlockMessageCellWidth                  = 55%
+PrefMailComposeRowSelectWidgetWidth            = 130px
+PrefMailComposeRowColorPickerWidgetWidth       = 1%
```

Comment:
Color Picker changes will be done in another ticket.
This changes here are to bind the color picker elements together.